MapboxNavigation/NavigationViewController.swift

```diff
@@ -331,19 +331,20 @@ open class NavigationViewController: UIViewController {
 
      See [Mapbox Directions](https://mapbox.github.io/mapbox-navigation-ios/directions/) for further information.
      */
-    @objc(initWithRoute:directions:styles:locationManager:)
+    @objc(initWithRoute:directions:styles:locationManager:voiceController:)
     required public init(for route: Route,
                          directions: Directions = Directions.shared,
                          styles: [Style]? = [DayStyle(), NightStyle()],
-                         locationManager: NavigationLocationManager? = NavigationLocationManager()) {
+                         locationManager: NavigationLocationManager? = NavigationLocationManager(),
+                         voiceController: RouteVoiceController? = nil) {
         
         super.init(nibName: nil, bundle: nil)
         
         self.routeController = RouteController(along: route, directions: directions, locationManager: locationManager ?? NavigationLocationManager())
         self.routeController.usesDefaultUserInterface = true
         self.routeController.delegate = self
         self.routeController.tunnelIntersectionManager.delegate = self
-        
+        self.voiceController = voiceController ?? MapboxVoiceController()
```

Comment:
Seems like this should follow the pattern of being declared in the initializer, like the others